platform/linuxbsd/wayland/wayland_thread.cpp

```diff
@@ -1596,13 +1594,13 @@ void WaylandThread::_wl_pointer_on_frame(void *data, struct wl_pointer *wl_point
 				if (test_button == MouseButton::WHEEL_UP || test_button == MouseButton::WHEEL_DOWN) {
 					// If this is a discrete scroll, specify how many "clicks" it did for this
 					// pointer frame.
-					mb->set_factor(abs(pd.discrete_scroll_vector.y));
+					mb->set_factor(fabs(pd.discrete_scroll_vector_120.y / (float)120));
```

Comment:
Prefer using our own `Math::abs` abstraction over the C API directly, this used to be a source of trouble on MSVC.